phabfive/cli.py

```diff
@@ -78,13 +78,20 @@
 sub_paste_args = """
 Usage:
     phabfive paste list
+    phabfive paste create <title> input <file> [options]
```

Comment:
I vore for removing `input` all together here, and just go with `phabfive paste create <title> <file>`. At least I can't see why not right now.